src/transformState.js

```diff
@@ -6,6 +6,39 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (let actionNumber = 0;
+    actionNumber < Object.keys(actions).length;
+    actionNumber++) {
```

Comment:
you dont need to use Object.keys here, just use the length of the actions, also, its a good practice to use `i` as an iterator variable name in the for loops